docs/rules/html-self-closing.md

```diff
@@ -0,0 +1,82 @@
+---
+pageClass: "rule-details"
+sidebarDepth: 0
+title: "svelte/html-self-closing"
+description: "enforce self-closing style"
+---
+
+# svelte/html-self-closing
+
+> enforce self-closing style
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> **_This rule has not been released yet._** </badge>
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+You can choose either two styles for elements without content
+
+- always: `<div />`
+- never: `<div></div>`
+
+<ESLintCodeBlock fix>
+
+<!-- prettier-ignore-start -->
+<!--eslint-skip-->
+
+```svelte
+<script>
+  /* eslint svelte/html-self-closing: "error" */
+</script>
+
+<!-- ✓ GOOD -->
+<div />
+<p>Hello</p>
+<div><div /></div>
+<img />
+<svelte:head />
+
+<!-- ✗ BAD -->
+<div></div>
+<p> </p>
+<div><div></div></div>
+<img>
+<svelte:head></svelte:head>
+```
+
+<!-- prettier-ignore-end -->
+
+</ESLintCodeBlock>
+
+## :wrench: Options
+
+```jsonc
+{
+  "svelte/html-self-closing": [
+    "error",
+    {
+      "html": {
+        "void": "always", // or "always" or "ignore"
+        "normal": "always", // or "never" or "ignore"
+        "component": "always", // or "never" or "ignore"
+        "svelte": "always" // or "never" or "ignore"
+      }
```

Comment:
I thought about this, but I think we can remove `html`. We probably don't plan to add options like `svg`. Also, I don't think there are any users who want to set them individually. What do you think?

```suggestion
      "void": "always", // or "always" or "ignore"
      "normal": "always", // or "never" or "ignore"
      "component": "always", // or "never" or "ignore"
      "svelte": "always" // or "never" or "ignore"
```

Reply:
Yeah, I don't know about anybody who would set that separately.

Reply:
Could you change it to omit the `html` field?

Reply:
yes, I didn't have time during the last 2 days. I'll change it now